nixos/modules/services/cluster/hadoop/yarn.nix

```
@@ -178,18 +178,18 @@ in
 
       services.hadoop.gatewayRole.enable = true;
 
-      services.hadoop.yarnSiteInternal = with cfg.yarn.nodemanager; {
-        "yarn.nodemanager.local-dirs" = localDir;
+      services.hadoop.yarnSiteInternal = with cfg.yarn.nodemanager; mkMerge [ ({
```

```suggestion
      services.hadoop.yarnSiteInternal = with cfg.yarn.nodemanager; mkMerge [
        {
```
and indent the following blocks

---

The style of indentation here is the same as it was before this PR. The commit in this PR fixes a bug in the way the attrsets are merged. I will be leaving the style of the code here unchanged in this PR.